test/extended/etcd/vertical_scaling.go

```diff
@@ -274,23 +256,169 @@ var _ = g.Describe("[sig-etcd][Feature:EtcdVerticalScaling][Suite:openshift/etcd
+	// 3) Delete the machine hosting the node in step 2.
+	// 4) Verify the member removal and the total voting member count of 2 to ensure scale-down happens first when a member is unhealthy.
+	// 5) Restore the initial cluster state by creating a new machine(scale-up) and re-enabling CPMS
+	g.It("is able to vertically scale down when a member is unhealthy [apigroup:machine.openshift.io]", func() {
+		framework.Logf("This test validates that scale-down happens before scale up if the deleted member is unhealthy")
+
+		etcdNamespace := "openshift-etcd"
+
+		if cpmsActive {
+			//step 0: disable the CPMS
+			framework.Logf("Disable the CPMS")
+			err := scalingtestinglibrary.DisableCPMS(ctx, g.GinkgoT(), cpmsClient)
+			err = errors.Wrap(err, "pre-test: failed to disable the CPMS")
+			o.Expect(err).ToNot(o.HaveOccurred())
+
+			// re-enable CPMS after the test
+			defer func() {
+				framework.Logf("Re-enable the CPMS")
+				err := scalingtestinglibrary.EnableCPMS(ctx, g.GinkgoT(), cpmsClient)
+				err = errors.Wrap(err, "post-test: failed to re-enable the CPMS")
+				o.Expect(err).ToNot(o.HaveOccurred())
+			}()
+		}
+
+		// test-setup: select a master machine to delete, then find its associated node and the etcd pod running on it
+		// pick a running master machine to be deleted
+		machineToDelete, err := scalingtestinglibrary.AnyRunningMasterMachine(ctx, g.GinkgoT(), machineClient)
+		err = errors.Wrap(err, "test-setup: failed to retrieve a running master machine for deletion")
+		o.Expect(err).ToNot(o.HaveOccurred())
+
+		// find the node hosted on the machine
+		etcdTargetNodeName := machineToDelete.Status.NodeRef.Name
+		o.Expect(etcdTargetNodeName).ToNot(o.BeNil(), fmt.Sprintf("test-setup: expected to find a NodeRef.Name for the machine %s, but it is nil", machineToDelete.Name))
+
+		etcdTargetNode, err := nodeClient.Get(ctx, etcdTargetNodeName, metav1.GetOptions{})
+		err = errors.Wrapf(err, "test-setup: failed to retrieve the node %s", etcdTargetNodeName)
+		o.Expect(err).ToNot(o.HaveOccurred())
+
+		// retrieve the pod on the etcdTargetNode that will be removed when the static pod manifest is moved from the node
+		etcdPods, err := kubeClient.CoreV1().Pods(etcdNamespace).List(ctx, metav1.ListOptions{LabelSelector: "app=etcd", FieldSelector: "spec.nodeName=" + etcdTargetNodeName})
+		err = errors.Wrapf(err, "test-setup: failed to retrieve the etcd pod on the node %s", etcdTargetNodeName)
+		o.Expect(err).ToNot(o.HaveOccurred())
+		etcdTargetPod := etcdPods.Items[0]
+
+		// step 1: make a member unhealthy by removing the etcd static pod manifest from the node, then stopping the kubelet on that node
+		framework.Logf("Removing the etcd static pod manifest from the node %s", etcdTargetNodeName)
+		err = oc.AsAdmin().Run("debug").Args("-n", etcdNamespace, "node/"+etcdTargetNodeName, "--", "chroot", "/host", "/bin/bash", "-c", "mkdir -p /var/lib/etcd-backup && mv /etc/kubernetes/manifests/etcd-pod.yaml /var/lib/etcd-backup").Execute()
```

Comment:
feel free to just call "rm". I think the backup folder just confuses other readers